salt/states/influxdb_user.py

```diff
@@ -52,11 +53,12 @@ def present(name,
         example user present in influxdb:
           influxdb_user.present:
             - name: example
-            - password: somepassword
+            - passwd: somepassword
             - admin: False
-            - grants:
-                foo_db: read
+            - grants: {
```

Comment:
Why not rely on yaml parsing for the dictionary here?

Reply:
As written it didn't work, I'm not super familiar with the way YAML parsing works though.